### PR TITLE
Cleanup tests to use "devnet" configuration files to run all tests - Closes #2156

### LINKED ISSUE #2156: Cleanup tests to use "devnet" configuration files to run all tests
### Expected behavior
Since we have network specific configurations in place, we should configure our tests to run against any of the configured network. And by default it should be run against `devnet` configuration. No need to manage seperate config files for tests. 

### Actual behavior
Currently we manage seperate config files only to run tests. 

### Which version(s) does this affect? (Environment, OS, etc...)
1.0.0

### PR DESCRIPTION
**Pull request by nazarhussain**
### What was the problem?
There were redundant configuration file, to be only used in running the tests. 

### How did I fix it?
Removed the redundant file and used the devnet configuration files. 

### How to test it?
Run all tests. 

### Review checklist

* The PR solves #2156
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

File: test/functional/system/blocks/chain/apply_block.js [hunk 1]
```diff
@@ -369,7 +368,7 @@ describe('system test (blocks) - chain/applyBlock', () => {
 					payloadHash:
 						'be0df321b1653c203226add63ac0d13b3411c2f4caf0a213566cbd39edb7ce3b',
 					payloadLength: 494,
-					previousBlock: genesisBlock.id,
+					previousBlock: __testContext.config.genesisBlock,
```
Comment (4miners) [inline test/functional/system/blocks/chain/apply_block.js]: Why not `__testContext.config.genesisBlock.id`?

File: test/integration/setup/network.js [hunk 2]
```diff
@@ -22,7 +22,7 @@ var utils = require('../utils');
 
 module.exports = {
 	waitForAllNodesToBeReady(configurations, cb) {
-		const retries = 20;
+		const retries = 40;
```
Comment (4miners) [inline test/integration/setup/network.js]: Why we need to increase that?

Comment (nazarhussain) [reply test/integration/setup/network.js]: This was messed up from me. I staged some changes, but whole file was committed.

**Comment by 4miners** [comment]
Great work @nazarhussain 👍

**Pull request merged**